sdk/compute/Azure.ResourceManager.Compute/tests/Generated/Samples/Sample_AvailabilitySetCollection.cs

```diff
@@ -0,0 +1,246 @@
+            // this example is just showing the usage of "AvailabilitySets_Get" operation, for the dependent resources, they will have to be created separately.
+
+            // authenticate your client
+            ArmClient client = new ArmClient(new DefaultAzureCredential());
+
+            // this example assumes you already have this ResourceGroupResource created on azure
+            // for more information of creating ResourceGroupResource, please refer to the document of ResourceGroupResource
+            string subscriptionId = "{subscription-id}";
+            string resourceGroupName = "rgcompute";
+            ResourceIdentifier resourceGroupResourceId = ResourceGroupResource.CreateResourceIdentifier(subscriptionId, resourceGroupName);
+            ResourceGroupResource resourceGroupResource = client.GetResourceGroupResource(resourceGroupResourceId);
+
+            // get the collection of this AvailabilitySetResource
+            AvailabilitySetCollection collection = resourceGroupResource.GetAvailabilitySets();
+
+            // invoke the operation
+            string availabilitySetName = "aaaaaaaaaaaa";
+            bool result = await collection.ExistsAsync(availabilitySetName);
+
+            Console.WriteLine($"Succeeded: {result}");
+        }
+
+        // AvailabilitySets_Get_MinimumSet_Gen
+        [NUnit.Framework.Test]
+        [NUnit.Framework.Ignore("Only verifying that the sample builds")]
+        public async Task Get_AvailabilitySetsGetMinimumSetGen()
+        {
+            // Generated from example definition: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-03-01/examples/availabilitySetExamples/AvailabilitySets_Get_MinimumSet_Gen.json
+            // this example is just showing the usage of "AvailabilitySets_Get" operation, for the dependent resources, they will have to be created separately.
+
+            // authenticate your client
+            ArmClient client = new ArmClient(new DefaultAzureCredential());
+
+            // this example assumes you already have this ResourceGroupResource created on azure
+            // for more information of creating ResourceGroupResource, please refer to the document of ResourceGroupResource
+            string subscriptionId = "{subscription-id}";
+            string resourceGroupName = "rgcompute";
+            ResourceIdentifier resourceGroupResourceId = ResourceGroupResource.CreateResourceIdentifier(subscriptionId, resourceGroupName);
+            ResourceGroupResource resourceGroupResource = client.GetResourceGroupResource(resourceGroupResourceId);
+
+            // get the collection of this AvailabilitySetResource
+            AvailabilitySetCollection collection = resourceGroupResource.GetAvailabilitySets();
+
+            // invoke the operation
+            string availabilitySetName = "aaaaaaaaaaaaaaaaaaaa";
```

Comment:
The main difference between `Get_AvailabilitySetsGetMinimumSetGen` and  `Get_AvailabilitySetsGetMaximumSetGen` is  the value of `availabilitySetName`, other code are basically the same. The original swagger samples probably have differences in the responses based on the example names. Can we config to suppress one of the two samples if we cannot check the duplication automatically?

Reply:
I prefer we do not check or suppress samples.
Samples will be show in the REST API doc page with the swagger example - when we have a swagger example, we should have a corresponding dotnet sample, otherwise the page viewer might have questions, like, why this example does not have its corresponding dotnet sample while other languages have?